var/spack/repos/builtin/packages/py-sina/no_orjson.patch

```diff
@@ -0,0 +1,15 @@
+--- a/python/setup.py
++++ b/python/setup.py
+@@ -51,9 +51,8 @@ setup(name='llnl-sina',
+         'six',
+         'sqlalchemy',
+         'enum34;python_version<"3.4"',
+-        'orjson;python_version>="3.6" and platform_machine!="ppc64le"',
+-        'ujson;python_version>="3.6" and platform_machine=="ppc64le"',
+-        'ujson<4;python_version<"3.6" and platform_machine!="ppc64le"',
++        'ujson;python_version>="3.6"',
++        'ujson<4;python_version<"3.6"',
```

Comment:
Let's just make it 'ujson' without the Python version check. This kind of stuff is only in there to workaround bugs in pip. Spack is smart enough to know that `py-ujson` 4+ only supports Python 3.

Reply:
Done.